src/object/cli_obj.c

```diff
@@ -1409,6 +1410,23 @@ obj_shards_2_fwtgts(struct dc_object *obj, uint32_t map_ver, uint8_t *bit_map,
 		return rc;
 	}
 
+	if (obj_auxi->spec_shard) {
+		D_ASSERT(grp_nr == 1);
+		D_ASSERT(shard_cnt == 1);
+		D_ASSERT(bit_map == NIL_BITMAP);
+		D_ASSERT(req_tgts->ort_srv_disp == 0);
+
+		req_tgts->ort_grp_nr = 1;
+		req_tgts->ort_grp_size = 1;
+		if (obj_is_ec(obj))
+			req_tgts->ort_start_shard = rounddown(start_shard, obj_get_grp_size(obj));
+
+		rc = obj_shard_tgts_query(obj, map_ver, start_shard,
```

Comment:
"rc = obj_shard_tgts_query(obj, map_ver, req_tgts->ort_start_shard" ?

Reply:
> Just aware of that the req_tgts->ort_grp_size is useless for this case, similarly for ort_grp_nr and ort_start_shard.

The following obj_req_fanout() and shard_io() will use these value.

Reply:
> "rc = obj_shard_tgts_query(obj, map_ver, req_tgts->ort_start_shard" ?

oh, ort_start_shard is the group start shard of the current group, where start_shard is.  And start_shard is the specify shard to be fetched/listed for this api, so it should use start_shard.